explorer/src/pages/Attestation/index.tsx

```diff
@@ -34,6 +37,7 @@ export const Attestation = () => {
 
   if (isLoading || isValidating) return <AttestationLoadingSkeleton />;
   if (!attestation) return <NotFoundPage id={id} page="attestation" />;
```

Comment:
I think that better to use some enum value for `page` prop.

Reply:
I use my type [Page](https://github.com/Consensys/linea-attestation-registry/blob/dev/explorer/src/interfaces/components/index.ts#L7)